_studio/shared/umc/codec/h265_dec/src/umc_h265_task_supplier.cpp

```diff
@@ -47,6 +47,18 @@
 namespace UMC_HEVC_DECODER
 {
 
+#if (MFX_VERSION >= 1025)
+inline void SetDecodeErrorTypes(NalUnitType nalUnit, mfxExtDecodeErrorReport *pDecodeErrorReport)
```

Comment:
I'd suggest to accept ```mfxExtDecodeErrorReport``` as reference, or you have to check pointer for null

Reply:
Will submit a new patch on this.

Reply:
I am new in GitHub MediaSDK, new patches on reviewed commit should submit as new commits or merge with original commit?

Reply:
@UngTengEn you need to amend your commit localy and force push it, so there should be only one single commit in history